Makefile

```diff
@@ -166,3 +166,10 @@ nightlybuild:
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_MINICLUSTER) -v ./tools/minicluster
 	export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):$(PWD)/crypto/lib:$(PWD)/crypto/lib/blslib
 	./bin/$(BUILD_TARGET_MINICLUSTER) -timeout=14400
+
+.PHONY: testcli
```

Comment:
This is added to test ioctl using a local bc server?

Reply:
Yes. It runs a bc server with different config.